apps/block_scout_web/lib/block_scout_web/controllers/api/v2/smart_contract_controller.ex

```diff
@@ -39,7 +39,7 @@ defmodule BlockScoutWeb.API.V2.SmartContractController do
            {:not_found, Chain.find_contract_address(address_hash, @smart_contract_address_options, false)} do
       conn
       |> put_status(200)
-      |> render(:smart_contract, %{address: address})
+      |> render(:smart_contract, %{address: address, conn: conn})
```

Comment:
It's not necessary to put connection. It is done automatically
```suggestion
      |> render(:smart_contract, %{address: address})
```

Reply:
Fixed in https://github.com/blockscout/blockscout/pull/9599/commits/3bdd76cbd93910f1c1f3378c02238a8f6396ced7